kie-server-parent/kie-server-api/src/build/revapi-config.json

```diff
@@ -0,0 +1,82 @@
+{
+  "revapi": {
+    "java": {
+      "filter": {
+        "_comment": "We don't want to check transitive classes, e.g. from OptaPlanner, since we already check them in their own module.",
+        "packages": {
+          "regex": true,
+          "include": [
+            "org\\.kie\\.server\\.api.*"
+          ]
+        }
+      }
+    },
+    "_comment": "Changes between 6.4.0.Final and master. These changes are desired and thus ignored. They should be removed when 7.0.0.Final is available.",
+    "ignore": [
+      {
+        "code": "java.method.returnTypeChanged",
+        "old": "method org.optaplanner.core.api.domain.solution.Solution org.kie.server.api.model.instance.SolverInstance::getBestSolution()",
+        "new": "method java.lang.Object org.kie.server.api.model.instance.SolverInstance::getBestSolution()",
+        "justification": "ADD YOUR EXPLANATION FOR THE NECESSITY OF THIS CHANGE"
+      },
+      {
+        "code": "java.method.returnTypeChanged",
+        "old": "method org.optaplanner.core.api.domain.solution.Solution org.kie.server.api.model.instance.SolverInstance::getPlanningProblem()",
+        "new": "method java.lang.Object org.kie.server.api.model.instance.SolverInstance::getPlanningProblem()",
+        "justification": "ADD YOUR EXPLANATION FOR THE NECESSITY OF THIS CHANGE"
+      },
+      {
+        "code": "java.method.removed",
+        "old": "method org.optaplanner.core.api.score.Score org.kie.server.api.model.instance.SolverInstance::getScore()",
+        "justification": "ADD YOUR EXPLANATION FOR THE NECESSITY OF THIS CHANGE"
+      },
+      {
+        "code": "java.method.parameterTypeChanged",
+        "old": "method parameter void org.kie.server.api.model.instance.SolverInstance::setBestSolution(===org.optaplanner.core.api.domain.solution.Solution===)",
+        "new": "method parameter void org.kie.server.api.model.instance.SolverInstance::setBestSolution(===java.lang.Object===)",
+        "justification": "ADD YOUR EXPLANATION FOR THE NECESSITY OF THIS CHANGE"
+      },
+      {
+        "code": "java.method.parameterTypeChanged",
+        "old": "method parameter void org.kie.server.api.model.instance.SolverInstance::setPlanningProblem(===org.optaplanner.core.api.domain.solution.Solution===)",
+        "new": "method parameter void org.kie.server.api.model.instance.SolverInstance::setPlanningProblem(===java.lang.Object===)",
+        "justification": "ADD YOUR EXPLANATION FOR THE NECESSITY OF THIS CHANGE"
+      },
+      {
+        "code": "java.method.removed",
+        "old": "method void org.kie.server.api.model.instance.SolverInstance::setScore(org.optaplanner.core.api.score.Score)",
+        "justification": "ADD YOUR EXPLANATION FOR THE NECESSITY OF THIS CHANGE"
```

Comment:
Moved to `org.kie.server.api.model.instance.SolverInstance.setScoreWrapper(org.kie.server.api.model.instance.ScoreWrapper)`